crates/sol-types/tests/ui/type.rs

```diff
@@ -202,7 +202,7 @@ sol! {
 
 sol! {
     struct CustomTypes {
-        bytes_ a;
+        // bytes_ a; inconsistent output
```

Comment:
what's going on here?

Reply:
`cargo t` vs `cargo t --workspace` have different output for the recursive error below, this one's fine